docs/stability-release.md

```diff
@@ -1,18 +1,46 @@
 # Release Process
 
-Wasmtime's release process was [originally designed in an RFC][rfc4] and this
-page is intended to serve as documentation for the current process as-is today.
+Wasmtime's release process was [originally designed in an RFC][rfc4] and later
+amended with [an LTS process][rfc-lts] and this page is intended to serve as
+documentation for the current process as-is today.
+
 The high-level summary of Wasmtime's release process is:
 
-* A new major version of Wasmtime will be made available once a month.
-* Security bugs and correctness fixes will be backported to the latest two
-  releases of Wasmtime and issued as patch releases.
+* A new major version of Wasmtime will be made available on the 20th of each
+  month.
+* Each release that is a multiple of 12 is considered an LTS release and is
+  supported for 24 months. Other releases are supported for 2 months.
+* Security bugs are guaranteed to be backported to all supported releases.
+* Bug fixes are backported on a volunteer basis.
+
+[rfc-lts]: https://github.com/bytecodealliance/rfcs/pull/42
+
+## Current Versions
+
+<div id='version-table'>
+
+This is a table of supported, recent, and some upcoming releases of Wasmtime
+along with the dates around their release process. Rows in **bold** are
+actively supported at this time.
+
+| Version    | LTS | Branch Date | Release Date | EOL Date |
+|------------|-----|-------------|--------------|----------|
+<!-- JS edits in the table -->
+
+In more visual form this is a gantt chart of the current release trains:
+
+<!-- JS edits in the chart -->
```

Comment:
An example of what this generates:

<img width="820" alt="Screenshot 2025-03-27 at 3 08 49 PM" src="https://github.com/user-attachments/assets/36845eaa-ebbb-4790-82b6-524f8b9536b3" />

Reply:
That is some impressive DOM-tree-editing hackery! The end result looks great.

At the risk of complexifying a little bit: could we add a placeholder ("view this document as HTML in a browser with JavaScript enabled to see contents" or somesuch) so that if the Markdown is processed in some other way -- for example, converted to print pdf by pandoc -- or viewed by a user with JS disabled, they aren't confused?

Reply:
Sounds good yeah, added some `<noscript>` tags to indicate that it's empty without JS